torch/csrc/jit/script/init.cpp

```diff
@@ -218,6 +218,15 @@ struct ModuleValue : public SugaredValue {
     return "module";
   }
 
+  at::Tensor* reverse_parameter_lookup(at::Tensor tensor) {
```

Comment:
This fails to check parameters of submodules, and I think having this functionality around is a bad idea.